scalafix-testkit/src/main/scala/scalafix/testkit/SyntacticRewriteSuite.scala

```diff
@@ -16,17 +15,14 @@ class SyntacticRewriteSuite(rewrite: Rewrite)
   def check(name: String, original: String, expected: String): Unit = {
     test(name) {
       import scala.meta._
-      val obtained =
-        Scalafix
-          .fix(Input.String(original), ScalafixConfig(rewrite = rewrite))
-          .get
+      val obtained = rewrite.apply(Input.String(original))
```

Comment:
This looks great!  Nothing left you can remove at this point without overloading rewrite.apply for strings vs files and given that can be contentious this seems about as close as you can possibly get to:

val obtained = rewrite(original)

Big fan of these changes 👍